docs/developer-manual/deployments.md

```diff
@@ -0,0 +1,107 @@
+---
+id: orcfax deployments
+sidebar_position: 3
+slug: /deployments
+---
+
+# Protocol deployments
+
+Signposting current and historical deployments of the Orcfax protocol. The
+hashes can be used to navigate Orcfax fact-statements and archival packages
+on Cardano and Arweave blockchains.
+
+This document serves as a reference for historical access to Orcfax data.
+Integrators should consult the [integrator docs][consume] for an explanation
+an the role of different scripts used in the protocol and how to correctly use
+the Orcfax protocol.
+
+## V1
+
+Orcfax V1 introduces the FSP (fact statement pointer) script. The FSP will not
+change in the lifetime of a deployment. The FSP is used to record the current
+FS (fact statement) script hash. The FSP, FS, and C (constitution) script
+hashes are documented below for each V1 deployment.
+
+### V1: Mainnet
+
+A summary of mainnet deployments is listed below.
+
+| Status | TYPE | FSP | FS | Arweave Hash |
+|--------|------|---- |----|--------------|
+| TBD | CER | `TBD` | `TBD` | TBD |
+
+#### Mainnet detailed overview
+
+A per-FSP breakdown is documented below.
+
+|                  | Details |
+|------------------|---------|
+| FSP              | -       |
+| Arweave Hash     | -       |
+| First appearance | -       |
+| Slot no.         | -       |
+| Block            | -       |
+| Last appearance  | -       |
+| Slot no.         | -       |
+| Block            | -       |
+
+### V1: Preview
+
+A summary of preview deployments is listed below.
+
+| Status | TYPE | FSP | FS | C | Arweave Hash |
+|--------|------|---- |----|---|--------------|
+| Active | CER | `0690081bc113f74e04640ea78a87d88abbd2f18831c44c4064524230` | `572f979076428566452f5c22bc98f58d49f237d4f53b86d37b147244` | `3a81e444b7b88e41d421551d056ce1e7701948236251019d6fdce656` | [6KymaAPWd3JNyMT0B7EPYij4TWxehhMrzRD8qifCSLs][arweave-preview] |
+
+<br/>
+
+#### Preview detailed overview
+
+A per-FSP breakdown is documented below.
+
+|                  | Details |
+|------------------|---------|
+| FSP              | `0690081bc113f74e04640ea78a87d88abbd2f18831c44c4064524230` |
+| Arweave Hash     | [6KymaAPWd3JNyMT0B7EPYij4TWxehhMrzRD8qifCSLs][arweave-preview] |
+| First appearance | 26.6.2024     |
```

Comment:
Can we stick to ISO 8601 (YYYY-MM-DD).
 for the sake of those unfortunate soles still thinking MM/DD/YYYY. Here clearly absurd, but generally not

Reply:
> Can we stick to ISO 8601 (YYYY-MM-DD).

Did you try to render it? (we have opted for dd.mm.yyyy because iSO doesn't render in the table markdown. Otherwise, it is the only standard I will use so this approach also pains me.

Reply:
![image](https://github.com/user-attachments/assets/a1fa8a8d-5f12-4f92-b3de-7e56ceea0f38)

Reply:
I see. I confused the issues, Docusaurus treats dates like this:

Reply:
I'm not particularly concerned on which punctuation mark.
`YYYY.MM.DD` is fine. 

The concern is around `DD.MM.YYYY` and `MM.DD.YYYY` (the latter is obvs an anathema, yet prevalent) 
are ambiguous in cases where the day happens to be `<13`